pkg/services/apiserver/options/extra.go

```diff
@@ -44,12 +44,17 @@ func (o *ExtraOptions) ApplyTo(c *genericapiserver.RecommendedConfig) error {
 	handler := slogadapter.New(log.New("grafana-apiserver"))
 	logger := slog.New(handler)
 	if err := utilfeature.DefaultMutableFeatureGate.SetFromMap(map[string]bool{
-		string(genericfeatures.APIServerTracing): false,
-		string(genericfeatures.WatchList):        true,
+		string(genericfeatures.WatchList): true,
 	}); err != nil {
 		return err
 	}
-	// if verbosity is 8+, response bodies will be logged. versboity of 7 should then be the max
+
+	// disabling configured trace provider
+	if c.TracerProvider != nil {
+		c.TracerProvider = nil
+	}
```

Comment:
@toddtreece  -- any sense what the right thing TODO is?

enterprise already has special tracing config

Reply:
@yuri-tceretian my guess at what was going on originally was that alerting uses different trace IDs or trace headers, and there is a conversion doesn't happen in the k8s tracing provider. does that sound correct? if so, do you happen to know how to test?